src/app/api/comments/route.ts

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+import { PrismaClient } from "@prisma/client";
+import { NextResponse, type NextRequest } from "next/server";
+import { createCommentSchema } from "./schema";
+
+const prisma = new PrismaClient();
```

Comment:
https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/nextjs-prisma-client-dev-practices

can you edit this to use the instance of prisma defined in lib/prisma.ts?

Reply:
solved